Misc/NEWS.d/next/Security/2024-10-31-13-14-27.gh-issue-126238.CZqaon.rst

```diff
@@ -0,0 +1 @@
+Fix a possible crash internally when compiling.
```

Comment:
I suggest to drop this NEWS entry.

Reply:
> drop this NEWS entry

Done )

Reply:
Why? This seems user-facing to me, isn't it?

Reply:
@sobolevn, @picnixz , what's the bottom line? Is news entry required or not?

Reply:
The bottom line in general is whether this kind of bug can be triggered easily using public interface. Unless you have a reproducer saying that with X and Y you can do Z, or unless the interface is publicly documented and known to the outside world, a NEWS entry would be fine. But here, we have neither a test nor is `_PyCompile_LookupArg` something that is exposed to the world.